tests/libnet/ipaddress.go

```diff
@@ -34,3 +37,19 @@ func getFamily(ip string) k8sv1.IPFamily {
 	}
 	return k8sv1.IPv4Protocol
 }
+
+func GetLoopbackAddress(family k8sv1.IPFamily) string {
+	if family == k8sv1.IPv4Protocol {
+		return "127.0.0.1"
+
+	}
+	return net.IPv6loopback.String()
+}
+
+func GetLoopbackAddressForUrl(family k8sv1.IPFamily) string {
+	address := GetLoopbackAddress(family)
+	if family == k8sv1.IPv6Protocol {
+		address = fmt.Sprintf("[%s]", address)
+	}
+	return address
+}
```

Comment:
Ipv4 address with brackets are legit too, we can just surround them with brackets ipv4 or ipv6 without checking